.travis.yml

```diff
@@ -7,7 +7,7 @@ rvm:
   - 2.2
   - 2.1
   - 2.0.0
-  - jruby-9.1.8.0
+  - jruby-9.1.10.0
```

Comment:
Please refer to: https://github.com/cucumber/cucumber-ruby/pull/1114#discussion_r118659144.